lib/connection/statement.js

```diff
@@ -359,6 +359,7 @@ function createContextPreExec(
   statementContext.multiCurId = statementOptions.multiCurId;
   statementContext.rowMode = statementOptions.rowMode;
   statementContext.asyncExec = statementOptions.asyncExec;
+  statementContext.cwd = statementOptions.cwd;
```

Comment:
What about assigning cwd to context in function 'createRowStatementPreExec'? It isn't needed?

Reply:
It isn't needed (I've applied this fix in the VS Code extension side as a patch), but are you recommending that I do for the sake of consistency?

Reply:
My recommendation is to add in `createRowStatementPreExec` as well.

Reply:
If it is performed in `createContextPreExec` isn't it already present in the context object? I can add it again but I'm a little confused on why it is needed. Let me know if I have misunderstood where to put it. I added it in `createRowStatementPreExec`.

Reply:
Yes, of course. You are right @sfc-gh-klin. It is a mistake in my comment. My worries was about 'createStatementPostExec'. This is method use for async queries and there the cwd parameter should be added to context as well. WDYT?